### ISSUE TITLE
Does not work in IE11

### ISSUE THREAD

**Issue by slavco86** [ai/nanoid#210]
This is what I get in IE11, and the value that should be assigned from nanoid is simply undefined
```ReferenceError: 'crypto' is undefined
   {
      [functions]: ,
      __proto__: { },
      description: "'crypto' is undefined",
      message: "'crypto' is undefined",
      name: "ReferenceError",
      number: -2146823279,
     ...
      Symbol()_6.o0bfewm05uz: undefined,
      Symbol()_g.o0bfewm05y8: undefined,
      Symbol(Symbol._hidden)_b.o0bfewm05y8: undefined,
      Symbol(Symbol.asyncIterator)_h.o0bfewm05y8: undefined,
      Symbol(Symbol.hasInstance)_c.o0bfewm05y8: undefined,
      Symbol(Symbol.match)_d.o0bfewm05y8: undefined,
      Symbol(Symbol.observable)_i.o0bfewm05y8: undefined,
      Symbol(Symbol.search)_e.o0bfewm05y8: undefined,
      Symbol(Symbol.split)_f.o0bfewm05y8: undefined
   }
```

possible solutions:
https://stackoverflow.com/questions/44042816/what-is-wrong-with-crypto-getrandomvalues-in-internet-explorer-11

**Comment by ai** [comment]
@slavco86 thanks for the report. We have IE support. Here we could have documentation or build tool issue.

1. How do you use Nano ID? Do you add `window.crypto` hack as [this error](https://github.com/ai/nanoid/blob/master/index.browser.js#L19) told you before?
2. How do you build your JS bundle? Do you use webpack? What is the webpack version?

**Comment by elliottsj** [comment]
I'm also experiencing an issue using this lib with IE11, but due to a syntax error in `index.browser.js`, on this line:

```js
let random = bytes => crypto.getRandomValues(new Uint8Array(bytes))
```

e.g. https://unpkg.com/nanoid@3.1.3/index.browser.js

This needs to be transpiled, because IE11 does not support `let` or arrows.

**Comment by ai** [comment]
@elliottsj you need to use Babel and compile `node_modules`. Check out our instructions in migration guide

https://github.com/ai/nanoid/releases/tag/3.0.0

**Comment by elliottsj** [comment]
Ah I missed that, thanks

**Comment by ai** [comment]
I improved IE docs 97cece9

**Comment by xuxucode** [comment]
Why not set a local `crypto` variable to `msCrypto` implicitly in this package, many other packages do this, see https://github.com/uuidjs/uuid/blob/master/src/rng-browser.js#L11.

I encounter an error when setting `window.crypto = window.msCrypto` in a Vue project:
```err
Uncaught TypeError: Cannot assign to read only property 'crypto' of object '#<Window>'
```
![image](https://user-images.githubusercontent.com/8274162/81072664-0dff2200-8f19-11ea-87c8-ebac0e5c1377.png)

**Comment by ai** [comment]
@xuxucode you need to change it only if it is not exist:

```js
if (!window.crypto) {
  window.crypto = window.msCrypto
}
```

I improved docs 4f9107b

**Comment by ricky11** [comment]
Hi, even after adding the conditional to a vue project, and using babel to transpile dependencies 'nanid'

i still get the warning 

![image](https://user-images.githubusercontent.com/5736415/88768615-e9c87f80-d1ad-11ea-851c-1b2235d0652b.png)

 if (!window.crypto) {

   window.crypto = window.msCrypto
 }
import {nanoid} from 'nanoid/'

the only way for me to solve this is import the nonsecure version in ie11. 

Any ideas.

**Comment by ai** [comment]
@ricky11 create a file `polyfills.js` with:

```js
if (!window.crypto) {
  window.crypto = window.msCrypto
}
```

and import it before `nanoid`:

```js
import './polyfills.js'
import {nanoid} from 'nanoid'
```

**Comment by ricky11** [comment]
Ahh... is it because the build process checks all the imports  first and does the conditional check later on..
So by creating a polyfill import , that will be sure to run before nonoid...

For now i am using /non-secure which as i understand does not hardware random bytes into the algorithm, what are the chances of collision for non-secure. the ID Collision Calculator doesn't show for nonsecure ?

Thanks for your amazing opensource software contributions.

**Comment by ai** [comment]
@ricky11 I improved docs 2ec4ea7

I released a new warning in 3.1.12.
